docs/concepts/abstractions/pod.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Pods
+---
+
+{% capture overview %}
+This page provides an overview of `Pod`, the smallest deployable object in the Kubernetes object model.
+{% endcapture %}
+
+* TOC
+{:toc}
+
+{% capture body %}
+### Understanding Pods
+
+A *Pod* is the basic building block of Kubernetes--the smallest and simplest unit in the Kubernetes object model that you create or deploy. A Pod represents a running process on your cluster.
+
+A Pod encapsulates an application container (or, in some cases, multiple containers), storage resources, a unique network IP, and options that govern how the container(s) should run. A Pod represents a unit of deployment: *a single instance of an application in Kubernetes*, which might consist of either a single container or a small number of containers that are tightly coupled and that share resources.
+
+> [Docker](https://www.docker.com) is the most common container runtime used in a Kubernetes Pod, but Pods support other container runtimes as well.
+
+Pods are employed a number of ways in a Kubernetes cluster, including:
+
+* **Pods that run a single container**. The "one-container-per-Pod" model is the most common Kubernetes use case; in this case, you can think of a Pod as a wrapper around a single container, and Kubernetes manages the Pods rather than the containers directly.
+* **Pods that run multiple containers that need to work together**. A Pod might encapsulate an application that relies on multiple co-located containers that are tightly coupled and need to share resources. These co-located containers might form a single cohesive unit of service--one container serving files from a shared volume to the public, while a separate "sidecar" container refreshes or updates those files. The Pod wraps these containers and storage resources together as a single managable entity.
```

Comment:
I think the word "compose" might be good in the description for multiple containers.  Composition is a well known technique in object oriented programming, and I think the mental model carries over.

WDYT of:

A Pod might encapsulate an application composed of several tightly coupled / co-located containers.  These containers may need to share local resources such as a disk or network loopback for communication.